docker/image/app/include/Dockerfile.base.twig

```diff
@@ -34,6 +39,13 @@ RUN [ -z "${SSH_DEPLOY_KEY:-}" ] || bash /ssh-keys.sh
 
 WORKDIR /app
 
+{% if @('pact.enabled') %}
+RUN gem install --no-document pact-provider-verifier pact_broker-client
+RUN apt-get remove -qq -y ruby-dev build-essential \
+    && apt-get auto-remove -qq -y \
+    && apt-get clean
+{% endif %}
```

Comment:
this needs to be in the same run command, otherwise there would be no reason to remove the packages

Reply:
yeah I thought that too, I will move it up